posts/2015-05-24-es6-gotchas.md

```diff
@@ -46,6 +46,21 @@ export default class ComponentName extends React.Component {
   }
 ~~~
 
+Another alternative syntax is to use the `::` ES7 [bind](https://github.com/zenparsing/es-function-bind)
+
+~~~js
+export default class ComponentName extends React.Component {
+
+  handleClick(e) => {
+    ...
+  }
+  
+  render(){
+    return (
+      <button onClick={::this.handleClick}> click </button> 
```

Comment:
Also, if you're using the => operator and property assignment, as you're doing in this example, the bind operator is not needed anyway. The function bind operator would get used if you were just doing a normal function definition in the class (as ES6 class functions don't get bound automatically). 

In general, I'm not sure it matters that much that doing "::" or ".bind(this)" within the render function creates a new function...the performance bottlenecks would be extremely minimal (in fact, binding in render is the route that FB recommends in their docs: https://facebook.github.io/react/docs/reusable-components.html#no-autobinding